src/gazebo_gps_plugin.cpp

```diff
@@ -104,7 +122,11 @@ void GpsPlugin::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf)
     gzmsg << "Home altitude is set to " << env_alt << ".\n";
```

Comment:
The msgs can be merged into the same message, outside of the if else conditions, printing the home position LLA.